### ISSUE TITLE
Official.fm ExternalUserPage 

### ISSUE THREAD

**Issue by pthulin** [youtify/youtify#190]

**Comment by pthulin** [comment]
From the official fm IRC channel:

5:01 PM <pthulin> hey dudes, how do i get info and about a user? I've tried http://api.official.fm/users/99006
5:02 PM <happydawn> pthulin, the api is in transition
5:04 PM <happydawn> the new api is going live next month I think
5:27 PM <happydawn> pthulin, there's no point in using the old API, it's deprecated and the new content is not accessible with it
5:30 PM <happydawn> pthulin, can the feature wait? I've already given Jens Nockert the necessary info to hack on the new API, he said he'd integrate it into youtfiy. I'm guessing he'll do it when he has a green light
5:31 PM <pthulin> sure, cool
5:31 PM <happydawn> pthulin, in any case the new API is not stable yet, so it's better to wait

Reassigning this to @JensNockert

**Comment by auroranockert** [comment]
Yeah, it's on my 'to-hack' list as soon as the new API is stable/public.

**Comment by crackofdusk** [comment]
The API has been made public, here are the docs: https://github.com/officialfm/api

**Comment by pthulin** [comment]
@JensNockert do you time to look at this? Official.fm shuts down their old API next week. Just say if you don't have the time and I'll do it!

/Per :)

**Comment by kallux** [comment]
I've already implemented the new api for search and play. Also I've started on the external profiles, so there's no need for you to do it.
